cypress/utils/utils.ts

```diff
@@ -828,22 +878,36 @@ export function createMultipleStakeholders(
     return stakeholdersList;
 }
 
+function generateRandomDateRange() {
```

Comment:
2 things here:

1. Can you `export` this function?
2. Could you add two optional date parameters so the user can provide a min and max ranges for the random generation?

Thanks!